src/App.js

```diff
@@ -91,15 +91,15 @@ class App extends Component {
   }
 
   getActiveSurvey = async () => {
-    if (this.props.auth0.isAuthenticated) {
-      const tokenResponse = await this.props.auth0.getIdTokenClaims();
-      const jwt = tokenResponse.__raw;
-
+    // if (this.props.auth0.isAuthenticated) {
+    //   const tokenResponse = await this.props.auth0.getIdTokenClaims();
+    //   const jwt = tokenResponse.__raw;
+    // }
```

Comment:
needed to remove auth0 from this request so the state is set on componentdidmount and an unauthorized user can take the survey.